assets/javascripts/discourse/initializers/extend-for-kolide.js.es6

```diff
@@ -1,9 +1,35 @@
 import { withPluginApi } from "discourse/lib/plugin-api";
 import I18n from "I18n";
+import { ajax } from "discourse/lib/ajax";
+import { popupAjaxError } from "discourse/lib/ajax-error";
 
 function initializeWithApi(api) {
   const currentUser = api.getCurrentUser();
 
+  function attachAssignButton(cooked) {
+    const buttons = cooked.querySelectorAll("a.kolide-assign") || [];
+
+    buttons.forEach((button) => {
+      button.addEventListener("click", assignUser, false);
+    });
+  }
+
+  function assignUser() {
+    const userId = this.dataset.user;
+    const deviceId = this.dataset.device;
+
+    ajax(`/kolide/devices/${deviceId}/assign.json`, {
+      type: "PUT",
+      data: { user_id: userId },
+    })
+      .then(() => {
+        bootbox.alert("Success");
+      })
+      .catch(popupAjaxError);
+
+    return false;
+  }
+
```

Comment:
Would be nice if there could be some component / acceptance tests. As I've not seen it before, I can't visualise how these things get triggered.

Reply:
One thing which would also be useful is something like [tgx's PR](https://github.com/discourse/discourse-question-answer/pull/18) where he inserts certain screenshots to allow the reviewer to understand what the code changes mean

Reply:
I added a screenshot in description 👍